packages/lib-classifier/src/workers/registerWorkers.js

```diff
@@ -0,0 +1,18 @@
+export default function registerWorkers () {
+  // Check that service workers are registered
+  console.log('Checking for service worker support')
+  if ('serviceWorker' in navigator) {
+    // Use the window load event to keep the page load performant
+    console.log('Service workers are supported')
+    window.addEventListener('load', () => {
```

Comment:
Would this worker ever need to be unbound? If so, a function declaration would be easier than an anonymous function

Reply:
I don't think we'll want to unbind the event listener on the window load event, but there's a chance we might want to unregister a worker, but probably not for this type that handles background sync. I came across an example use case for unregistering for workers that handle asset caching, but not for background sync. The worker itself manages reinstallation when its done with its current jobs and there's a new version of the file or can be configured to not wait and install immediately.

Anyway here's the overview on the basics for the workbox implementation: https://developers.google.com/web/tools/workbox/guides/get-started